components/eventing-controller/pkg/backend/utils/eventmesh_utils_test.go

```diff
@@ -61,55 +62,48 @@ func TestConvertKymaSubToEventMeshSub(t *testing.T) {
 	expectedWebhookURL := fmt.Sprintf("%s://%s", scheme, host)
 	g := NewGomegaWithT(t)
 
-	// @TODO: Fix this test after update protocol settings for EventMesh
-	//t.Run("subscription with protocol settings where defaults are overridden", func(t *testing.T) {
-	//	// given
-	//	subscription := eventingtestingv2.NewSubscription("name", "namespace",
-	//		eventingtestingv2.WithDefaultSource(),
-	//		eventingtestingv2.WithOrderCreatedFilter(),
-	//		eventingtestingv2.WithValidSink("ns", svcName),
-	//	)
-	//
-	//	subscription.Spec.ProtocolSettings = eventingtestingv2.NewProtocolSettings(
-	//		eventingtestingv2.WithBinaryContentMode(),
-	//		eventingtestingv2.WithExemptHandshake(),
-	//		eventingtestingv2.WithAtLeastOnceQOS(),
-	//		eventingtestingv2.WithDefaultWebhookAuth(),
-	//	)
-	//
-	//	eventTypeInfos := getTypeInfos(subscription.Spec.Types)
-	//
-	//	// Values should be overridden by the given values in subscription
-	//	expectedWebhookAuth := &types.WebhookAuth{
-	//		Type:         types.AuthTypeClientCredentials,
-	//		GrantType:    types.GrantTypeClientCredentials,
-	//		ClientID:     subscription.Spec.ProtocolSettings.WebhookAuth.ClientID,
-	//		ClientSecret: subscription.Spec.ProtocolSettings.WebhookAuth.ClientSecret,
-	//		TokenURL:     subscription.Spec.ProtocolSettings.WebhookAuth.TokenURL,
-	//	}
-	//	expectedBEBSubscription := eventingtestingv2.NewBEBSubscription(
-	//		defaultNameMapper.MapSubscriptionName(subscription.Name, subscription.Namespace),
-	//		*subscription.Spec.ProtocolSettings.ContentMode,
-	//		expectedWebhookURL,
-	//		bebSubEvents,
-	//		expectedWebhookAuth,
-	//	)
-	//
-	//	apiRule := eventingtestingv2.NewAPIRule(subscription,
-	//		eventingtestingv2.WithPath(),
-	//		eventingtestingv2.WithService(svcName, host),
-	//	)
-	//
-	//	// then
-	//	gotBEBSubscription, err := ConvertKymaSubToEventMeshSub(
-	//		subscription, eventTypeInfos, apiRule, defaultWebhookAuth,
-	//		defaultProtocolSettings, defaultNamespace, defaultNameMapper,
-	//	)
-	//
-	//	// when
-	//	g.Expect(err).To(BeNil())
-	//	g.Expect(*expectedBEBSubscription).To(Equal(*gotBEBSubscription))
-	//})
+	t.Run("subscription with protocol settings and webhook auth", func(t *testing.T) {
```

Comment:
(nit) There are multiple `t.Run` here, does it make sense to use table tests here?

Reply:
changed it to table tests